libs/zzz/formula/src/util.ts

```diff
@@ -174,3 +175,14 @@ export function teamData(members: readonly Member[]): TagMapNodeEntries {
     members.map((src) => teamEntry.add(reader.withTag({ src, et: 'own' }))),
   ].flat()
 }
+
+export function getFormula(sheet: Sheet | undefined, name: string | undefined) {
+  if (!sheet || !name) return
+  return (formulas as any)[sheet]?.[name] as
```

Comment:
The whole thing is typed, why are we making it so jank?

Reply:
it didn't play nicely dynamically, and I didn't bother spending the time to make it pretty.